docs/llm/quickstart.md

```diff
@@ -38,7 +38,7 @@ node: {
   node_options: {
       [type.googleapis.com / mediapipe.LLMCalculatorOptions]: {
           models_path: "./",
-          plugin_config: '{"KV_CACHE_PRECISION": "u8", "DYNAMIC_QUANTIZATION_GROUP_SIZE": "32"}',
+          plugin_config: '\''{"KV_CACHE_PRECISION": "u8", "DYNAMIC_QUANTIZATION_GROUP_SIZE": "32"}'\'',
```

Comment:
why this is changed? It will not work in my opinion.

Reply:
Escape character was added inside echo '_<json_content>_' command.
Is ' character required in output file?
This character was used in graph.pbtxt in continuous batching demo: https://github.com/openvinotoolkit/model_server/blob/main/demos/continuous_batching/graph.pbtxt (line 19)